lib/routes/routePOST.js

```diff
@@ -4,6 +4,8 @@ import api from '../api/api';
 import routesUtils from './routesUtils';
 import pushMetrics from '../utilities/pushMetrics';
 
+const MAX_POST_LENGTH = 1024 * 1024; // 1 MB
```

Comment:
It was. Updated in https://github.com/scality/S3/pull/567/commits/0d52ada141e7743b76e80347b3a2c5c6908acd43.